.pipelines/nightly.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
       # - KEYVAULT_SECRET_NAME
       - group: e2e-environment-variables
       - name: REGISTRY
-        value: upstreamk8sci.azurecr.io/azure-workload-identity
```

Comment:
what tag are we using? can someone overwrite this?

Reply:
We are using https://github.com/Azure/azure-workload-identity/blob/main/Makefile#L19-L21.

Reply:
Image version is overwritten at build time - https://github.com/Azure/azure-workload-identity/blob/da5aeeadda58eba6cb904468f392b05288b300e1/scripts/ci-e2e.sh#L15